[quickfix] Build error

### Operating system

Linux

### Compiler

GCC 12.1.0

### Steps to reproduce the behavior

```Shell
./vcpkg install quickfix
```


### Failure logs

Package: quickfix[core]:x64-linux -> 1.15.1#8

**Host Environment**

- Host: x64-linux
- Compiler: GNU 12.1.0
-    vcpkg-tool version: 2023-04-07-bedcba5172f5e4b91caac660ab7afe92c27a9895
    vcpkg-scripts version: 512d62f7e 2023-05-09 (17 hours ago)

**To Reproduce**

`vcpkg install quickfix`

**Failure logs**

```
-- Using cached quickfix-quickfix-v1.15.1.tar.gz.
-- Cleaning sources at /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /zdata/libs/vcpkg/downloads/quickfix-quickfix-v1.15.1.tar.gz
-- Applying patch 00001-fix-build.patch
-- Applying patch fix_wsl_symlink_error.patch
-- Using source at /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /zdata/libs/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j9
    Working Directory: /zdata/libs/vcpkg/buildtrees/quickfix/x64-linux-dbg
    See logs for more information:
      /zdata/libs/vcpkg/buildtrees/quickfix/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/quickfix/portfile.cmake:41 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/zdata/libs/vcpkg/buildtrees/quickfix/install-x64-linux-dbg-out.log</summary>

```
[1/55] cd /zdata/libs/vcpkg/buildtrees/quickfix/x64-linux-dbg && /zdata/libs/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E make_directory /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/include/
[2/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FieldTypes.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FieldTypes.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FieldTypes.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldTypes.cpp
[3/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Dictionary.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Dictionary.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Dictionary.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Dictionary.h:31,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Dictionary.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:26:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[4/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FieldConvertors.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.cpp:27:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:26:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[5/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FieldMap.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[6/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Group.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Group.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Group.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Group.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/Group.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Group.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Group.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Group.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Group.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Group.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Group.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[7/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionaryProvider.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/DataDictionaryProvider.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionaryProvider.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FixFields.h:4,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Fields.h:25,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionary.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionaryProvider.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionaryProvider.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[8/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileLog.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FileLog.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileLog.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Message.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Log.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileLog.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileLog.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[9/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionary.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/DataDictionary.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionary.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FixFields.h:4,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Fields.h:25,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionary.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/DataDictionary.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[10/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Acceptor.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/Acceptor.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Acceptor.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Message.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Application.h:25,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Acceptor.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Acceptor.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
[11/55] /usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileStore.cpp
FAILED: src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o 
/usr/bin/c++ -DGETHOSTBYNAME_R_INPUTS_RESULT=1 -DHAVE_SSL=1 -DHAVE_STD_SHARED_PTR -DHAVE_STD_SHARED_PTR=1 -DHAVE_STD_UNIQUE_PTR -DHAVE_STD_UNIQUE_PTR=1 -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++ -I/zdata/libs/vcpkg/installed/x64-linux/include -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean -I/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src -fPIC -g -MD -MT src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o -MF src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o.d -o src/C++/CMakeFiles/quickfix.dir/FileStore.cpp.o -c /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileStore.cpp
In file included from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldConvertors.h:30,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:32,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FieldMap.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Message.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/MessageStore.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileStore.h:29,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/FileStore.cpp:26:
/zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Exceptions.h:39:3: error: looser exception specification on overriding virtual function ‘virtual FIX::Exception::~Exception() noexcept (false)’
   39 |   ~Exception() QUICKFIX_THROW() {}
      |   ^
In file included from /usr/include/c++/12/system_error:41,
                 from /usr/include/c++/12/bits/ios_base.h:46,
                 from /usr/include/c++/12/ios:42,
                 from /usr/include/c++/12/istream:38,
                 from /usr/include/c++/12/sstream:38,
                 from /zdata/libs/vcpkg/buildtrees/quickfix/src/v1.15.1-294be70ca5.clean/src/C++/Field.h:29:
/usr/include/c++/12/stdexcept:138:13: note: overridden function is ‘virtual std::logic_error::~logic_error() noexcept’
  138 |     virtual ~logic_error() _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW;
      |             ^
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

### Additional context

_No response_

--- a/ports/quickfix/00001-fix-build.patch
+++ b/ports/quickfix/00001-fix-build.patch
@@ -75,7 +75,7 @@ index 0aa2cd3e..2829e110 100644
  #endif
  
 +#ifdef __cpp_noexcept_function_type
-+#define QUICKFIX_THROW(...) noexcept(false)
++#define QUICKFIX_THROW(...) noexcept
 +#else
 +#define QUICKFIX_THROW(...) throw(__VA_ARGS__)
 +#endif

--- a/ports/quickfix/vcpkg.json
+++ b/ports/quickfix/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "quickfix",
   "version": "1.15.1",
-  "port-version": 8,
+  "port-version": 9,
   "description": "QuickFIX is a free and open source implementation of the FIX protocol.",
   "homepage": "https://github.com/quickfix/quickfix",
   "supports": "!uwp & !(osx & arm64)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6922,7 +6922,7 @@
     },
     "quickfix": {
       "baseline": "1.15.1",
-      "port-version": 8
+      "port-version": 9
     },
     "quill": {
       "baseline": "2.9.1",

--- a/versions/q-/quickfix.json
+++ b/versions/q-/quickfix.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "08f8b2eb6b5493a1faae9707021fe1d287ed6fa5",
+      "version": "1.15.1",
+      "port-version": 9
+    },
+    {
       "git-tree": "a8cdc0c05fb161bb28059f93d8dc05ea60d4c118",
       "version": "1.15.1",
       "port-version": 8

